README.md

```diff
@@ -1,2 +1,243 @@
-A co-working space located at 16 Palmers Rd, London, E2 0SY, United Kingdom
+
+> _Where London's most creative people go to get (["deep"](https://www.youtube.com/watch?v=KX_2a_jsGYw)) work **done**_.
+
+![quiet-please-wall-art](https://cloud.githubusercontent.com/assets/194400/23511482/1cbe7720-ff55-11e6-8f21-82753e5dab6a.jpg)
+
+## _Why_?
+
+There are _hundreds_ of co-working spaces/offices in London,
+so why do we **_need_ another one**?
+
+> `As a` ***creative person*** (_writer/designer/developer/videographer_)<br />
+> `I need` a **_quite_ place** to get my **work _done_** (_without interruptions_) <br />
+> `So that` I can ***be/feel productive***. <br />
+
+> As the members of QuietSpace want to _share_ that environment with other like-minded creative people.
+
+### `QuietSpace`'s _Mission_ is to curate the most <br />_creatively productive_ working environment in London.
+
+## What?
+
+### A Co-Working Space Located at 16 Palmers Rd, London, E2 0SY
+
+Map: https://www.google.co.uk/maps/place/16+Palmers+Rd,+London+E2+0SY/ <br />
+Contact: <a href="tel:02036090609">0203 609 0609</a>
+
+#### Features
+
++ A well-designed work space with all the amenities you expect (_good internet and cell phone signal_)
+
++ Proximity to transport links (Tube & Bus), "Cycle Hire" station 100m from our door.
+
++ Multiple bicycle racks (_for locking your bike_) 50m from office.
+
++ Close to town if you need to go in for a client meeting, pitch or lunch with a friend.
+
++ Like-minded creative professionals working on highly innovative projects. We share all our knowledge openly/cooperatively so you can learn how to make any digital product from concept/design to running on customer's devices!
+
++ Next to Regents Canal (_perfect for people who cycle or like running_)
+
++ 4 Parks within walking distance.
+
++ Great Gym (2 minutes walk)
+
++ Out-door/Park Gym (_free. also 2 mins walk_)
+
++ Mile End Climbing Wall (5 minutes walk): https://www.mileendwall.org.uk/
+
++ Great restaurants & cafes nearby 5mins walk.
+
++ Free tea & coffee (_tell us what your favourite brand/flavour is and will make sure it's always in stock!_)
+
++ Free Fresh Fruit for members.
+
++ Blender and Juicer equipment so you can make your own. (_or for "cost price" you can buy made-this-moring cold-pressed organic juice or smoothies from our supplier_)
+
++ Fridge and microwave oven.
+
+#### Desk Rental Rates >> _Simple/Transparent Pricing_
+
+We aim to be _cost-effective_ while giving our members lots of added value.
+
++ Monthly Subscription **£180/desk** (_hot desk + clean desk policy_)
++ Pay-as-you-Go **£11/day** (_sold in blocks of 5 "credits"_)
+
+> "Old" Pricing: http://www.founderlondon.com/images/workhere.jpg <br />
+> Discussion: https://github.com/foundersandcoders/london-programme/issues/42
+
+### World-Class Meetup & Training Venue
+
+Part dwyl's of our mission to help everyone discover what they love doing requires us to share our knowledge through Meetups and training.
```

Comment:
Typo: 'part dwyl' :arrow_right: 'part of dwyl'

Also, is this the first time we mention dwyl? We need to say above this somewhere that QuietSpace is run by dwyl.